### DISCUSSION TITLE
A screen where we can view all the logs.

### DISCUSSION THREAD

**Discussion by mehmetuken** [coollabsio/coolify#1787] [Feature Requests]
A screen where we can view all container logs and perform container-based filtering would be excellent. Instead of navigating through all projects to see logs individually, we could monitor the entire event status from that screen. I'm currently using drain logs, but having a screen where we can also view logs from Coolify for quick interventions would be beneficial.

**Comment by codefromhimanshu** [comment]
Can we use grafana or Dozzle for this usecase?
